packages/import/src/importIssues.ts

```diff
@@ -272,7 +272,7 @@ export const importIssues = async (apiKey: string, importer: Importer): Promise<
 
     await client.issueCreate({
       teamId,
-      projectId: (projectId as unknown) as string,
```

Comment:
Looks like these are valid prettier changes based on the config. Think we are good to merge @rlargman